service/start-acs.sh

```diff
@@ -38,7 +38,7 @@ main() {
         esac
     done
 
-    cp "${DIR}"/target/acs-service-*.jar "${DIR}"/.acs-service-copy.jar
+    cp "${DIR}"/target/acs-service-*-exec.jar "${DIR}"/.acs-service-copy.jar
```

Comment:
Placeholder comment to make sure to remember to make the necessary changes to acs-ci-config branch and jenkins because of this change